File: 백준/Bronze/34414. Tall Enough/Tall Enough.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int elem,n = scan.nextInt();
        boolean rlt=true;
        for(int i=0;i<n;i++) {
            elem=scan.nextInt();
            if(elem<48) {
                rlt=false;
                break;
            }
        }
        System.out.println(rlt?"True":"False");
    }
}

File: 백준/Bronze/34414. Tall Enough/Tall Enough.cc
#include <iostream>
using namespace std;
int main(void) {
    int n,elem;
    bool rlt=true;
    cin>>n;
    for(int i=0;i<n;i++) {
        cin>>elem;
        if(elem<48) {
            rlt=false;
            break;
        }
    }
    cout<<(rlt?"True":"False");
    return 0;
}